libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/headline/headlineBucketConversion.ts

```diff
@@ -0,0 +1,59 @@
+// (C) 2023 GoodData Corporation
+import {
+    IInsightToPropConversion,
+    PropMeta,
+    sdkModelPropMetas,
+} from "../../../utils/embeddingCodeGenerator/index.js";
+import { bucketMeasure, bucketMeasures, IBucket, IMeasure, insightBucket } from "@gooddata/sdk-model";
+import { IEmbeddingCodeContext } from "../../../interfaces/VisualizationDescriptor.js";
+
+export function singleSecondaryMeasureBucketConversion<TProps extends object, TPropKey extends keyof TProps>(
+    propName: TPropKey,
+    bucketName: string,
+): IInsightToPropConversion<TProps, TPropKey, IMeasure> {
+    return bucketConversion(
+        propName,
+        sdkModelPropMetas.Measure.Single,
+        bucketName,
+        (ctx) => {
+            return !ctx?.settings?.enableNewHeadline;
+        },
```

Comment:
Just a small nitpick for this block and the one on line 33 in this file:
If you would write it as `(ctx) => !ctx?.settings?.enableNewHeadline`, you will save two lines, four lines in total in this file ;)